testbed/stack.ts

```diff
@@ -11,8 +11,8 @@ export interface TestbedProps extends cdk.StackProps {
 }
 
 export class Testbed extends cdk.Stack {
-    constructor(scope: cdk.Construct, id: string, props: TestbedProps) {
-        super(scope, id)
+    constructor(scope: cdk.Construct, id: string="testbed", props: TestbedProps) {
```

Comment:
Do the defaulting at the outer level.

Reply:
I thought defaulting in the constructor is the way to go about it.